views_uid2/index.hbs

```diff
@@ -1,258 +1,424 @@
-        visibility: hidden !important;
-      }
+    .anima- input:focus {
+      outline: none;
+    }
 
-      .anima-listeners-active-click,
-      .anima-listeners-active-click * {
-        cursor: pointer;
-      }
-    </style>
-  </head>
-  <body style="margin: 0; background: rgba(255, 255, 255, 1)">
-    <input type="hidden" id="anPageName" name="page" value="desktop-1" />
-    <div class="anima-container-center-horizontal">
-      <div class="desktop-1 anima-screen">
-        <div class="welcome-to-160766-C61RwL anima-valign-text-middle raleway-bold-stratos-14px">{{__ "Welcome to"}}</div>
-        <div class="ad-transpa-com-160769-C61RwL anima-valign-text-middle raleway-bold-stratos-24px">
-            {{__ "Transparency and Control Portal"}}
+    .anima-component-wrapper,
+    .anima-component-wrapper * {
+      pointer-events: none;
+    }
+
+    .anima-component-wrapper a *,
+    .anima-component-wrapper a,
+    .anima-component-wrapper input,
+    .anima-component-wrapper video,
+    .anima-component-wrapper iframe,
+    .anima-listeners-active,
+    .anima-listeners-active * {
+      pointer-events: auto;
+    }
+
+    .anima-hidden,
+    .anima-hidden * {
+      visibility: hidden;
+      pointer-events: none;
+    }
+
+    .anima-smart-layers-pointers,
+    .anima-smart-layers-pointers * {
+      pointer-events: auto;
+      visibility: visible;
+    }
+
+    .anima-component-wrapper.anima-not-ready,
+    .anima-component-wrapper.anima-not-ready * {
+      visibility: hidden !important;
+    }
+
+    .anima-listeners-active-click,
+    .anima-listeners-active-click * {
+      cursor: pointer;
+    }
+
+    /* select with CSS avatar icons */
+    option.avatar {
+      background-repeat: no-repeat !important;
+      padding-left: 20px;
+    }
+
+    .avatar .ui-icon {
+      background-position: left top;
+    }
+
+    .ui-selectmenu-button.ui-button {
+      flex: 1;
+      width: 2em !important;
+      background: none;
+      border: 1px solid black;
+      padding: 7px;
+      margin-right: 2px;
+    }
+
+    .ui-selectmenu-menu .ui-menu.avatar .ui-menu-item-wrapper {
+      padding: 2px 10px 0 30px;
+      color: var(--stratos);
+      font-family: 'Raleway', Helvetica, Arial, serif;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 500;
+    }
+
+    .ui-selectmenu-menu .ui-menu.avatar .ui-menu-item .ui-icon {
+      height: 24px;
+      width: 24px;
+      top: 0.1em;
+    }
+
+    .ui-selectmenu-text {
+      padding-left: 3em;
+      margin-right: 0px;
+      color: var(--stratos);
+      font-family: 'Raleway', Helvetica, Arial, serif;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 500;
+    }
+
+    .avatar-selected {
+      position: absolute;
+      height: 24px;
+      width: 24px;
+      right: auto;
+      margin-top: -8px;
+      top: 50%;
+      background-repeat: no-repeat;
+    }
+
+    .ui-button .ui-icon {
+      width: 0px;
+      background-image: none;
+    }
```

Comment:
This should probably get removed as not having an icon here is confusing to a user.

```suggestion
```